lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -174,6 +185,15 @@ export default function ContentSelector({
               Select PDF from Canvas
             </LabeledButton>
           )}
+          {blackboardEnabled && (
+            <LabeledButton
+              onClick={() => selectDialog('blackboardLmsFile')}
+              variant="primary"
+              data-testid="lms-file-button"
+            >
+              Select PDF from Blackboard
```

Comment:
This works for a proof of concept. For a production implementation I think we'll want to use the same button / dialog type for Canvas and Blackboard files but change the `listFilesApi` prop and button label according to the LMS being used.